packages/progress-indicator/src/LionProgressIndicator.js

```diff
@@ -0,0 +1,110 @@
+import { svg, css, LitElement } from '@lion/core';
+import { localize, LocalizeMixin } from '@lion/localize';
+
+export class LionProgressIndicator extends LocalizeMixin(LitElement) {
+  static get styles() {
+    return [
+      css`
+        :host {
+          display: inline-block;
+        }
+
+        svg.lion-progress {
```

Comment:
I have to add a class specifier to the svg tag in order to avoid naming collision with extensions of this component to be safe. I don't particularly like that

Reply:
We don't use `lion` in the classname, so if you fully override the styles you don't need to keep calling it `lion-`. Just use `.progress-indicator`.

Reply:
Removed the styling altogether